src/utils/language/nb.ts

```diff
@@ -16,4 +16,15 @@ export const nb = {
   tagExPublished: 'EksPublisert',
   tagNonEditable: 'Ikke redigerbar',
   tagUnderReview: 'Til høring',
+
+  dcatKeywords: {
+    must: 'Obligatorisk',
+    should: 'Anbefalt',
+    may: ''
```

Comment:
skal dette feltet være tom?

Reply:
Ja, var meningen. I designet er de valgfrie feltene ('may') ikke markert med noe, så var for å markere det. Men entryen kan like godt fjernes.